components/Nav.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import { Row, Col, Spacer, Select } from '@geist-ui/react';
+import { Sun, Moon } from '@geist-ui/react-icons';
+
+const darkLogo = '/images/opticDarkLogo.png';
+const lightLogo = '/images/opticLightLogo.png';
+
+const NavBar = styled(Row)`
+  align-items: center !important;
+  height: 50px;
+  border-bottom: 0.5px solid;
+  margin-bottom: 15px;
+  padding: 0px 15px;
+`;
+
+const Logo = styled.img`
+  height: 40px;
+`;
+
+const SelectContainer = styled(Col)`
+  text-align: right;
+`;
+
+interface NavParams {
+  setUseLight(useLight: boolean): any;
+  useLight: boolean;
+}
+
+const Nav = ({ useLight, setUseLight }: NavParams): JSX.Element => {
+  return (
+    <NavBar>
+      <Col>
+        <Logo src={useLight ? darkLogo : lightLogo} alt="OpticPower" />
+      </Col>
+      <SelectContainer>
+        <Select placeholder={useLight ? 'Light Mode' : 'Dark Mode'}>
+          <Select.Option value="Dark Mode" onClick={(): void => setUseLight(false)}>
```

Comment:
nit: why `onClick` on each option instead of `onChange` on the Select? 

I had used something like:
```js
<Select placeholder={useLight ? 'Light Mode' : 'Dark Mode'} onChange={(ev) => setLight(ev.target.value)}>
  <Select.Option value="false">
    <Moon size={16} /> <Spacer inline x={0.35} />
    Dark Mode
  </Select.Option>
  {/* ... */}
</Select>
```

Just curious.

Reply:
Hey, @lardissone I was trying your suggestion but I believe that you do that on MaterialUI, and works perfectly. If that is the case the prop `value` on MUI it can be any type. But on `react-geist`, the type of `value` can only be a string. The only way I was able to make this work was by doing the following:

```js
<Select placeholder={useLight ? 'Light Mode' : 'Dark Mode'} onChange={value => setUseLight(value === 'true')}>
          <Select.Option value="false">
            <Moon size={16} /> <Spacer inline x={0.35} />
            Dark Mode
          </Select.Option>
          <Select.Option value="true">
            <Sun size={16} /> <Spacer inline x={0.35} />
            Light Mode
          </Select.Option>
 </Select>
```

Let me know what you think.